source/language/classical.rst

```diff
@@ -110,6 +111,43 @@ multiplication, and division and the corresponding assignment operators.
    angle[10] c;
    c = angle(a + b); // cast to angle[10]
 
+
+Evaluation order
+~~~~~~~~~~~~~~~~
+
+OpenQASM evaluates expressions from left to right.
+
+   .. table:: [operator-precedence] operator precedence in OpenQASM.
+
+      +-------------------------------+---------------------------------------+
+      | Operator                      | Operator Types                        |
+      +-------------------------------+---------------------------------------+
+      | (), [], ++, (type)(x)         | Call, index, incrementors, cast       |
+      +-------------------------------+---------------------------------------+
+      | !, -, ~                       | Unary                                 |
+      +-------------------------------+---------------------------------------+
+      | *, /, %                       | Multiplicative                        |
+      +-------------------------------+---------------------------------------+
+      | +, -                          | Additive                              |
+      +-------------------------------+---------------------------------------+
+      | <<, >>                        | Bit Shift                             |
+      +-------------------------------+---------------------------------------+
+      | <, <=, >, >=                  | Comparison                            |
+      +-------------------------------+---------------------------------------+
+      | !=, ==                        | Equality                              |
+      +-------------------------------+---------------------------------------+
+      | &                             | Bitwise AND                           |
+      +-------------------------------+---------------------------------------+
+      | ^                             | Bitwise XOR                           |
+      +-------------------------------+---------------------------------------+
+      | |                             | Bitwise OR                            |
```

Comment:
The `|` seems to not show up in the table? May need to write it as a special char (or maybe this is an error in markdown and it will appear in sphinx)